Mathlib/Data/Set/Pairwise/Lattice.lean

```diff
@@ -55,6 +55,14 @@ theorem pairwiseDisjoint_sUnion {s : Set (Set ι)} (h : DirectedOn (· ⊆ ·) s
     (⋃₀ s).PairwiseDisjoint f ↔ ∀ ⦃a⦄, a ∈ s → Set.PairwiseDisjoint a f :=
   pairwise_sUnion h
 
+theorem pairwiseDisjoint_union_of_disjoint
```

Comment:
Could you add a docstring to this lemma? The statement is a little bit hard to parse.